src/performance/mania/ManiaScore.cpp

```diff
@@ -58,72 +57,64 @@ void ManiaScore::computeTotalValue()
 		multiplier *= 0.50f;
 
 	_totalValue =
-			std::pow(
-			std::pow(_strainValue, 1.1f) +
-			std::pow(_accValue, 1.1f), 1.0f / 1.1f
-		) * multiplier;
+		std::pow(
+			std::pow(_difficultyValue, 1.1f) +
+				std::pow(_accuracyValue, 1.1f),
```

Comment:
weird formatting

Reply:
This is how VSCode formatted it ;__; Just rolled with it instead of opening up vim and doing manual changes.